Tools/Taxonomy/Cwe/CweSchema.cs

```diff
@@ -2588,69 +2591,188 @@ public enum TechnologyNameEnumeration {
+    StorageHardware,
+    
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Test/Debug Hardware")]
+    TestDebugHardware,
+    
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Accelerator IP Hardware")]
+    AcceleratorIPHardware,
+    
+    /// <remarks/>
+    Accelerator,
     
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Analog and Mixed Signal IP")]
     AnalogandMixedSignalIP,
     
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Analog and Mixed Signal")]
+    AnalogandMixedSignal,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Audio/Video IP")]
     AudioVideoIP,
     
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Audio/Video")]
+    AudioVideo,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Bus/Interface IP")]
     BusInterfaceIP,
     
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Bus/Interface")]
+    BusInterface,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Clock/Counter IP")]
     ClockCounterIP,
     
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Clock/Counter")]
+    ClockCounter,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Communication IP")]
     CommunicationIP,
     
+    /// <remarks/>
+    Communication,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Controller IP")]
     ControllerIP,
     
+    /// <remarks/>
+    Controller,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Memory IP")]
     MemoryIP,
     
+    /// <remarks/>
+    Memory,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Microcontroller IP")]
     MicrocontrollerIP,
     
+    /// <remarks/>
+    Microcontroller,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Network on Chip IP")]
     NetworkonChipIP,
     
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Network on Chip")]
+    NetworkonChip,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Power Management IP")]
     PowerManagementIP,
     
+    /// <remarks/>
+    [System.Xml.Serialization.XmlEnumAttribute("Power Management")]
+    PowerManagement,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Processor IP")]
     ProcessorIP,
     
+    /// <remarks/>
+    Processor,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Security IP")]
     SecurityIP,
     
+    /// <remarks/>
+    Security,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Sensor IP")]
     SensorIP,
     
+    /// <remarks/>
+    Sensor,
+    
     /// <remarks/>
     [System.Xml.Serialization.XmlEnumAttribute("Storage IP")]
     StorageIP,
     
+    /// <remarks/>
+    Storage,
```

Comment:
Is there a reason these don't have an XmlEnumAttribute but the rest do?

Reply:
My understanding is this Attribute is needed when need it different form the var name, like need a space or slash, which is not valid in var names. If the same it is omitted.

This file is auto generated.